src/clients/printer_auto.rb

```diff
@@ -318,9 +319,7 @@ def ReadFileContent(file_name)
       # there is no string "]]>" (except perhaps in a comment).
       # It is o.k. to ignore when the sed command fails because then
       # the file content was (hopefully) not changed at all which is the right fallback:
-      Printerlib.ExecuteBashCommand(
-        Ops.add("sed -i -e 's/]]>/] ]>/g' ", file_name)
-      )
+      Printerlib.ExecuteBashCommand("sed -i -e 's/]]>/] ]>/g' " + file_name.shellescape)
```

Comment:
absolute path to sed? when you touch it?

Reply:
See general comments above